packages/integrations/image/README.md

```diff
@@ -186,17 +200,21 @@ A `number` can also be provided, useful when the aspect ratio is calculated at b
 
 Source for the original image file.
 
-For images in your project's repository, use the path relative to the `src` or `public` directory. For remote images, provide the full URL.
+For images located in your project's `src`: use the file path relative to the `src` directory. (e.g. `src="../assets/source-pic.png"`)
+
+For images located in your `public` directory: use the URL path relative to the `public` directory. (e.g. `src="/images/public-image.jpg"`)
+
+For remote images, provide the full URL. (e.g. `src="https://astro.build/assets/blog/astro-1-release-update.avif"`)
 
 #### alt
 
 <p>
 
 **Type:** `string`<br>
-**Default:** `undefined`
+**Required:** `true`
 </p>
 
-If provided, the `alt` string will be included on the built `<img />` element.
+The `alt` string will be included on the built `<img />` element.
```

Comment:
add a note here about why required, how to use `alt=""` if its decorative, etc. You can probably copy-paste this from the note in the README